reframe/frontend/cli.py

```diff
@@ -32,6 +32,31 @@
 from reframe.frontend.printer import PrettyPrinter
 
 
+REFRAME_ENV = [
```

Comment:
I also found these RFM_* variables, should they be added?
```
RFM_MODULES_PRELOAD
RFM_RECURSIVE
RFM_TIMESTAMP
```
If you took the list from the manpage maybe it should be updated there as well.

Reply:
Yes I got from the manpage

Reply:
I don't like this duplication. This mapping is stored already inside the `ArgumentParser`. Also we could instead print the configuration parameters as stored in the site configuration and mark in parenthesis where did the configuration parameter got its value. NB: I don't know how easy is this currently, but we should investigate. I do believe that this duplication can be avoided.

Reply:
@vkarak, I see that it is possible to retrieve the env variables from the argparser._options_map. Should I proceed like that? Or see the alternative that you are saying?

Reply:
@teojgo I think you should just print the environment variables. If somebody needs to check the exact value of a configuration option, she can use the `--show-config` option. If we also fix #1318 we will have the complete information needed to debug. As for the debug output regarding this, I'd suggest to print also the environment variables that are not set by showing them as `<not set>`, e.g., `RFM_PREFIX=<not set>`

Reply:
And also remove the current "Command-line configuration" debug message.